http/server_hash.go

```diff
@@ -0,0 +1,14 @@
+package http
+
+import (
+	"github.com/OneOfOne/xxhash"
+	"fmt"
+)
+
+func GetServerHash(url string) string{
+	return fmt.Sprintf("%x", xxhash.ChecksumString64(url))
+}
+
+func GetServerHashS(url string, database string) string{
```

Comment:
Can we rename this to something more descriptive? I don't understand what's that S in the end means.